ckanext/ontario_theme/fanstatic/common.css

```diff
@@ -1081,6 +1193,143 @@ via submission form. Copied from DS.*/
     line-height: 1.5;
   }
 }
+.ministry-contact {
+	margin-top: 28px;
+	margin-bottom: 28px;
+}
+
+.resource-item {
+  padding-top: 15px;
+	padding-bottom: 15px;
+}
+
+.resource-item-description{
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 24.51px;
+  color: #1a1a1a;
+}
+.label.label-english,
+.label.label-french,
+.label.label-english_and_french {
+  background-color: #f2f2f2;
+  font-size: 20px;
+  font-weight: 700;
+  color: #1a1a1a;
+  border-radius: 2px;
+  margin-left: 10px;
+  margin-bottom: 5px;
+}
+
+.resource-item-title{
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 27px;
+  color: #000000;
+  text-decoration: none;
+}
+
+.resource-item-title:hover{
+  color: #000000;
+}
+
+.dataset-font{
+  font-size: 19px;
+  line-height: 25.6px;
+  font-weight: 400;
+}
+.right-col{
+  float: right;
+  max-width: 350px;
+}
+.aside-heading {
+  color: #4d4d4d;
+  font-size: 19px;
+  line-height: 25.6px;
+  font-weight: 600;
+}
+
+.right-col h3 {
+  color: #000000;
+}
+
+.right-col a.btn.btn-primary{
+  font-size: 20px;
+  line-height: 27.24px;
+  font-weight: 700;
+}
+
+.right-col .module.module-narrow.module-shallow{
+  padding: 30px;
+}
+.package-z-index{
+  z-index: 2;
+  position: relative;
+}
+
+section.additional-info table.table {
+  border: none;
+}
+section.additional-info table.table thead {
+  display: none;
+}
+section.additional-info table.table td {
+  border: none;
+}
+section.additional-info table.table tbody tr:nth-child(even) th,
+section.additional-info table.table tbody tr:nth-child(even) td {
+  background-color: #f9f9f9;
+}
+section.additional-info table.table tbody tr th:first-child,
+section.additional-info table.table tbody tr td:first-child{
+  padding-top: 10px;
+}
+section.additional-info table.table tbody tr th:last-child,
```

Comment:
This bottom border is not actually being added.

Reply:
Actually none of the bottom borders in the th column are equal to the td bottom borders